monitors/SyntheticPegMonitor.js

```diff
@@ -77,8 +77,8 @@ class SyntheticPegMonitor {
       this.logger.warn({
         at: "SyntheticPegMonitor",
         message: "Unable to get price",
-        uniswapTokenPrice: uniswapTokenPrice ? uniswapTokenPrice.toString() : null,
-        cryptoWatchTokenPrice: cryptoWatchTokenPrice ? cryptoWatchTokenPrice.toString() : null
```

Comment:
I think this was causing either `uniswapTokenPrice` or `cryptoWatchTokenPrice` to not appear in the logs because of how Slack logs print out null objects? Not sure but I modified this to be safe

Reply:
+1, this makes sense.